.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+<!-- If you have an issue with a plugin consider creating an issue in the plugin's github repo instead. -->
+
+<!-- Provide a general summary in the Title above -->
+
+### Expected Behavior
+<!-- If you're describing a bug, tell us what should happen -->
+<!-- If you're suggesting a change/improvement, tell us how it should work -->
+
+### Current Behavior
+<!-- If describing a bug, tell us what happens instead of the expected behavior -->
+<!-- If suggesting a change/improvement, explain the difference from current behavior -->
+
+### Steps to Reproduce (for bugs)
+<!-- Please add a series of steps to reproduce the problem. -->
+
+### Versions
+<!-- Please run commands and include versions below. -->
+- **Homebridge**: `homebridge --version`
+- **Node**:       `node --version`
+- **OS**:         <!-- The version and architecture of your operating system -->
+
+### Configuration
+<!-- If relevant, include the contents of your config.json file between the two ``` lines below. -->
```

Comment:
I might encourage folks to only include the relevant portion of the config since it could be quite large - and emphasize to remove things like passwords :)

Reply:
It says "Please extract only the relevant portions if possible" already (is that enough?). But I can add about removing any sensitive information.

Reply:
Oh you mean for config, nevermind, I'll add